Test/Invoke-WhiskeyTask.Tests.ps1

```diff
@@ -157,6 +174,33 @@ function GivenDefaults
     $script:taskDefaults[$ForTask] = $Default
 }
 
+function GivenScmBranch
+{
+    param(
+        [string]
+        $Branch
+    )
+    $script:scmBranch = $Branch
+}
+
+function GivenToolInstalledTo
+{
+    param(
+    )
+
+    Mock -CommandName 'Install-WhiskeyTool' -ModuleName 'Whiskey'
+}
+
+function GivenVariable
+{
+    param(
+        $Name,
+        $Value
+    )
+
+    $variables[$Name] = $Value
+}

```

Comment:
You only need to scope when assigning a value to a variable. Since I'm setting a value in the hashtable, but not the hashtable itself, and there is no local `$variables` variable, PowerShell uses the script-level variable.

Reply:
Oh that's right. I overlooked that we're simply setting a value inside the hashtable.